connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -68,7 +65,6 @@
 
 @RunWith(PowerMockRunner.class)
 public class WorkerSourceTaskTest extends ThreadedTest {
```

Comment:
Can we just drop the `ThreadedTest` now? We no longer have a `ShutdownableThread` which is what `ThreadedTest` relies on.